source/docs/dashboard.md

```diff
@@ -0,0 +1,20 @@
+title: Working with the dashboard
+---
+
+![Dashboard](http://i.imgur.com/s4OQZpu.jpg)
```

Comment:
This image should live in the repo.

Reply:
Agreed. Also as mentioned in https://github.com/embark-framework/embark-site/pull/82#discussion_r221361496 I think we should swap it out with a more "neutral" version of it.